### PR TITLE
docs(README.md): toturial of open airflow page in browser

### PR DESCRIPTION
**Pull request by owem0410**
## Types of changes

- **Documentation Update**
toturial of open airflow page in browser

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [x] Run `poetry run pytest` locally to ensure all linter checks pass
- [x] Update the documentation if necessary

### CODE REVIEW

**Review by tai271828 — APPROVED** [review]
Soft +1.

It's not very conventional to place a png file in the root folder as a source code project. Something like `docs` folder is considerable. However, our project is not that big and many png and doc files exist.

While I am ok to approve the pull request, if @owem0410 could put the png in a `docs` folder, it will be better.

**Comment by tai271828** [comment]
@owem0410  you may want to edit your pull request description more carefully, for instance, removing the template messages and reserve only the part of what your pull request is.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ Using Airflow to implement our ETL pipelines
     * production: `docker run --rm -p 80:8080 --name airflow  -v $(pwd)/dags:/usr/local/airflow/dags -v $(pwd)/service-account.json:/usr/local/airflow/service-account.json --env-file=./.env.production davidtnfsh/pycon_etl:prod webserver`
     * dev/test: `docker run --rm -p 80:8080 --name airflow  -v $(pwd)/dags:/usr/local/airflow/dags -v $(pwd)/service-account.json:/usr/local/airflow/service-account.json --env-file=./.env.staging davidtnfsh/pycon_etl:test webserver`
     * Note the difference are just the env file name and the image cache.
+4. Enter the `localhost:80` in the address bar in your browser. Open the Airflow Page.
```
Comment (uranusjr) [inline README.md:56]: Is `:80` needed here? Web browsers default to the 80 port.

**Comment by tai271828** [comment]
@owem0410 did you forget to delete the png file in root while you have added `docs/airflow.png`. If the pull request is still in progress, you may "Convert to draft" (see the button on top-right corner of this page).

**Comment by owem0410** [comment]
> Soft +1.
> 
> It's not very conventional to place a png file in the root folder as a source code project. Something like `docs` folder is considerable. However, our project is not that big and many png and doc files exist.
> 
> While I am ok to approve the pull request, if @owem0410 could put the png in a `docs` folder, it will be better.

**Comment by owem0410** [comment]
> @owem0410 did you forget to delete the png file in root while you have added `docs/airflow.png`. If the pull request is still in progress, you may "Convert to draft" (see the button on top-right corner of this page).

Sorry, I'm unfamiliar with github. I will Convert to draft first.

File: README.md (hunk 1 above)
Comment (owem0410) [reply README.md:56]: Yes , not needed. Thank you.
I forget the default port is 80 for HTTP.

**Review by david30907d — APPROVED** [review]

**Pull request closed without merging**